README.md

```diff
@@ -220,3 +224,34 @@ This may be better suited for you if you plan to change some of the automaticall
 ## Recap
 
 After following this complete README, we should have an ETH smart contract as well as a Starknet smart contract, both connected to act as a bridge between these two chains.
+
+## Upgrade Contracts in Testnet
+
+### Starknet
+
+If you want to upgrade a previously deployed `Escrow` contract, it is possible through a command. We will perform the upgrade using `starkli`, so the same configuration used for deployment is necessary.
+
+Keep in mind that this command will **rebuild** `Escrow.cairo`, perform the **declare**, and call the external **upgrade()** function with the new class hash resulting from the **declare**. 
+
+To be able to upgrade the contract, you must be the **owner** of the contract and set the `ESCROW_CONTRACT_ADDRESS` variable in the `.env` file with the address of the `Escrow` contract that you want to be **upgraded**.
```

Comment:
I would maybe clarify that the ESCROW_CONTRACT_ADDRESS is set automatically by deploy.sh

Reply:
I added this comment just below 

```
.. # other envs
   ESCROW_CONTRACT_ADDRESS = You can either set an escrow address manually or run deploy.sh, and it will be set automatically
```

I could put it there as well, but I think it would be too repetitive. And while it's true that it's automatically set with the execution of deploy.sh, it's not the only way.